src/collection/views/Collection.tsx

```diff
@@ -141,19 +105,22 @@ const Collection: FunctionComponent<CollectionProps> = ({ match, history }) => {
 					text: collection.description,
 				} as BlockIntroProps,
 			});
-			(collection.collection_fragments || []).forEach(
-				(collectionFragment: Avo.Collection.Fragment) => {
-					contentBlockInfos.push({
-						blockType: 'VideoTitleTextButton',
-						content: {
-							title: collectionFragment.custom_title,
-							text: collectionFragment.custom_description,
-							videoSource: '',
-							buttonLabel: 'Meer lezen',
-						} as BlockVideoTitleTextButtonProps,
-					});
-				}
-			);
+
+			const fragments = orderBy([...collection.collection_fragments], 'position', 'asc') || [];
+
+			fragments.forEach((collectionFragment: Avo.Collection.Fragment) => {
+				console.log(collectionFragment);
+				contentBlockInfos.push({
+					blockType: collectionFragment.external_id ? 'VideoTitleTextButton' : 'RichText',
+					content: {
+						title: collectionFragment.custom_title,
+						text: collectionFragment.custom_description,
+						titleLink: `/${RouteParts.Item}/${collectionFragment.external_id}`,
```

Comment:
Use generateContentLinkString helper:
https://github.com/viaacode/avo2-client/blob/master/src/shared/helpers/generateLink.tsx#L68

mogelijks moet je de functie uitbreiden met een optional param: type: item | collection

Reply:
Heb dit per ongeluk in een andere PR gefixt.